cmake/projects/dbus/hunter.cmake

```diff
@@ -0,0 +1,27 @@
+if(DEFINED HUNTER_CMAKE_PROJECTS_DBUS_HUNTER_CMAKE_)
+  return()
+else()
+  set(HUNTER_CMAKE_PROJECTS_DBUS_HUNTER_CMAKE_ 1)
+endif()
+
+include(hunter_add_version)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME dbus
+    VERSION "1.10.0-hunter-1"
+    URL "https://github.com/daminetreg/dbus/archive/dbus-1.10.0-hunter-1.tar.gz"
+    SHA1 67123664d253ac0a63fd5f678624a20415258059 
+)
+
+hunter_cmake_args(
+    dbus 
+    CMAKE_ARGS
+      DBUS_ENABLE_XML_DOCS=OFF 
+)
+
+hunter_pick_scheme(DEFAULT url_sha1_cmake)
+hunter_download(PACKAGE_NAME dbus
+  PACKAGE_DEPENDS_ON expat)
```

Comment:
@daminetreg This will not download `expat` package but only register it. I've update documentation to clarify this: https://github.com/ruslo/hunter/wiki/dev.modules#hunter_download

I've got an error while running `hunter_add_package(dbus)` only:

```
-- Check size of socklen_t
-- Check size of socklen_t - done
CMake Error at cmake/CMakeLists.txt:346 (message):
  expat not found!
```

Note that if `expat` will be installed system-wide it **will be** found. It's not good and may result conflicts, see [wiki](https://github.com/ruslo/hunter/wiki/usr.adding.new.package.with.dependencies).

By the was since `dbus` has an example already I think you can create separate pull request with only `dbus` package so we can avoid discussion of unrelated issues.